app/assets/stylesheets/govuk_publishing_components/components/_highlight-boxes.scss

```diff
@@ -2,56 +2,56 @@
   @include govuk-text-colour;
   display: flex;
   flex-wrap: wrap;
-  margin: $gutter-half 0 0 0;
+  margin: govuk-spacing(3) 0 0 0;
   padding: 0;
 
-  @include media(tablet) {
+  @include govuk-media-query($from: tablet) {
     margin-right: -25px;
   }
 }
 
 .gem-c-highlight-boxes__item-wrapper {
-  @include core-19;
-  @include box-sizing(border-box);
+  @include govuk-font(19);
+  box-sizing: border-box;
   list-style-type: none;
-  padding: ($gutter / 6) $gutter-one-third ($gutter-half + $gutter-one-third) 0;
+  padding: (govuk-spacing(6) / 6) govuk-spacing(2) (govuk-spacing(3) + govuk-spacing(2)) 0;
```

Comment:
Shouldn't `$gutter-half + $gutter-one-third` be replaced by `govuk-spacing(5)`?

(similar instances of this below)